packages/styles/_typography.scss

```diff
@@ -3,9 +3,9 @@
 /* Regular (Sans) Font Sizes
 ========================================================================== */
 // For each size in the $type-sizes variable
-// Create class (.xxl)
+// Create class (.type-xxl)
 @each $name, $size in $type-sizes {
-  .#{$name} {
+  .type-#{$name} {
```

Comment:
this is a breaking change, do any kongponents use this class? Do we use it in kong-admin or khcp?

Reply:
no the only type references in either repo are in css and not class names